templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListItem/list-item.ts

```diff
@@ -5,6 +5,7 @@
   templateUrl: './list-item.component.html'
 })
 export class ListItemComponent implements OnInit {
+  // tslint:disable-next-line
```

Comment:
Quick question, why is this needed?

Reply:
Good question! So, the backend returns a list item that has attributes `text` and `_id`. When we make HTTP request to get the list data from the backend, it is assumed to be of type 'IListItem'. We cannot/should not change the data returned from the backend and if we modify the 'IListItem' in front-end to have `text` & `id` instead of `text` & `_id`, the data returned from the backend would break.

So, in that case, I thought we should keep the attribute `_id` as is and disable the lint for this one. Do you have any other suggestions?

Reply:
It sounds reasonable!